### ISSUE TITLE
Add Bootstrap Icons to NuGet

### ISSUE THREAD

**Issue by simo026q** [twbs/icons#1475]
I would love to have Bootstrap Icons as a NuGet package.

**If I don't want to use the CDN I have to go through all these steps when I need to update it:**
1. Download the font from the source code
2. Download the style from the CDN
3. Change the URL of the font in the style
4. Add it to my project

It would save me a lot of time if I only had to click update in the NuGet package manager.

**Comment by vanillajonathan** [comment]
It is already available on https://www.npmjs.com/package/bootstrap-icons

Super easy to install:

    npm i bootstrap-icons

**Comment by simo026q** [comment]
> It is already available on https://www.npmjs.com/package/bootstrap-icons

Yeah, but I am not aware of any support for npm in ASP.NET, hence why I want to use NuGet

**Comment by vanillajonathan** [comment]
npm works great with ASP.NET and ASP.NET Core. If you use Visual Studio you can use the Visual Studio Installer to install node.js with npm or you can install it from [nodejs.org](https://nodejs.org).
Then just open a command prompt and type `npm init` and `npm i bootstrap-icons`.

Visual Studio also have another way to manage client-side libraries, just right-click on your project and select "Manage Client-Side Libraries..." in the menu and it will create a JSON file for you. See https://learn.microsoft.com/en-us/aspnet/core/client-side/libman/

**Comment by XhmikosR** [comment]
We don't plan to publish on NuGet to keep things simple.
